https/README.md

```diff
@@ -0,0 +1,24 @@
+# HTTPS Package for Prometheus
+
+The `https` directory contains a Go package and a sample configuration file for running `node_exporter` with HTTPS instead of HTTP.
+When running a server with TLS use the flag `--web.tls-config`
+
+e.g. `./node_exporter --web.tls-config="https/tls-config.yml"`
```

Comment:
Aren't we planning on doing basic auth too? A more generic flag name would be better

Reply:
`https` doesn't cover basic auth.